pr-data.csv

```diff
@@ -1824,24 +1824,24 @@ https://github.com/vmware/admiral,e4b02936cc7d4ff2714e7231db0c4373ba5d48a2,compu
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testEnumWithJsonValueMethodAnnotation,ID,Accepted,https://github.com/vojtechhabarta/typescript-generator/pull/677,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testEnumUsingToString,ID,,,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testEnumWithJsonPropertyAnnotations,ID,,,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testEnumWithJsonValueFieldAnnotation,ID,,,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testEnumWithJsonValueMethodAnnotation,ID,,,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testObjectEnum,ID,,,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testSingleEnum,ID,Accepted,https://github.com/vojtechhabarta/typescript-generator/pull/677,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.EnumTest.testSingleEnum,ID,,,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.ext.OnePossiblePropertyValueAssigningExtensionTest.testGeneration,ID,,,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.ext.RequiredPropertyConstructorExtensionTest.testEnums,ID,,,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.FullyQualifiedNamesTest.test,ID,,,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.GenericsTest.testAdvancedGenerics,ID,,,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.ImmutablesTest.testImmutables,ID,Accepted,https://github.com/vojtechhabarta/typescript-generator/pull/686,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.ImmutablesTest.testImmutables,ID,,,
```

Comment:
Please do not remove the PR links and Status of these lines (vojtechhabarta/typescript-generator)